ccd-definition/CaseField.json

```diff
@@ -1139,5 +1139,14 @@
     "Label": "Supplementary evidence handled",
     "FieldType": "YesOrNo",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "sendToCtsc",
+    "Label": " ",
+    "FieldType": "YesOrNo",
+    "SecurityClassification": "Public",
+    "Comment": "Flag to support wether case has been sent to CTSC"
```

Comment:
```suggestion
    "Comment": "Flag to support whether case has been sent to CTSC"
```

Reply:
I would actually suggest this is the wrong comment as well. This field describes whether a case SHOULD be sent to CTSC. Maybe even add the value of the move date?